src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java

```diff
@@ -394,7 +394,8 @@ private Screen autoAdvanceSession(
                 try {
                     doPostAndSync((FormplayerSyncScreen) nextScreen);
                 } catch (SyncRestoreException e) {
-                    throw new CommCareSessionException(e.getMessage(), e);
+                    // ignore for now to keep smart links working
+                    // throw new CommCareSessionException(e.getMessage(), e);
```

Comment:
@orangejenny @shubham1g5 FYI I'm ignoring case claim exceptions here to make smart links work. See https://github.com/dimagi/commcare-hq/pull/31738

Reply:
cool, makes sense